source/upgrade.txt

```diff
@@ -27,7 +27,7 @@ Before you upgrade, perform the following actions:
   application runs on. See the :ref:`<node-compatibility>`
   page for this information.
 - Address any breaking changes between the version of the driver
-  your application currently uses and your planned upgrade version in the
+  your application uses now and your planned upgrade version in the
```

Comment:
```suggestion
  your application uses and your planned upgrade version in the
```

Reply:
I'm going to leave this one - I think it emphasises the comparison.